### PR TITLE
ApiCacheRepository now logs all its major steps.

### PR DESCRIPTION
**Pull request by sergii-vostrikov**
Resolves: OLPEDGE-528

Signed-off-by: Sergii Vostrikov <ext-sergii.vostrikov@here.com>

### CODE REVIEW

File: olp-cpp-sdk-dataservice-read/src/repositories/ApiCacheRepository.cpp [hunk 1]
```diff
@@ -20,8 +20,11 @@
 #include "ApiCacheRepository.h"
 
 #include <olp/core/cache/KeyValueCache.h>
+#include <olp/core/logging/Log.h>
 
 namespace {
+constexpr auto kLogtag = "ApiCacheRepository";
```
Comment (andescu) [inline olp-cpp-sdk-dataservice-read/src/repositories/ApiCacheRepository.cpp]:
```suggestion
constexpr auto kLogTag = "ApiCacheRepository";
```

File: olp-cpp-sdk-dataservice-read/src/repositories/ApiCacheRepository.cpp [hunk 2]
```diff
@@ -41,15 +44,18 @@ void ApiCacheRepository::Put(const std::string& service,
                              const std::string& serviceVersion,
                              const std::string& serviceUrl) {
   std::string hrn(hrn_.ToCatalogHRNString());
+  auto key = CreateKey(hrn, service, serviceVersion);
+  LOG_TRACE_F(kLogtag, "Put '%s'", key.c_str());
```
Comment (andescu) [inline olp-cpp-sdk-dataservice-read/src/repositories/ApiCacheRepository.cpp]: Why trace and not info?
Same below.

Comment (sergii-vostrikov) [reply olp-cpp-sdk-dataservice-read/src/repositories/ApiCacheRepository.cpp]: as per decision to put methods entry at trace level

Comment (andescu) [reply olp-cpp-sdk-dataservice-read/src/repositories/ApiCacheRepository.cpp]: Methods entry yes but you are printing parameters.
Method entry trace log should be without parameters print:
E.g.
```
LOG_TRACE_F(kLogTag, "%s", __PRETTY_FUNCTION__);
```

**Pull request merged**